### PR TITLE
Language fixes

### PR DESCRIPTION
**Pull request by dmach**
This is related to https://www.redhat.com/en/blog/update-red-hats-conscious-language-efforts
I've renamed several variables which actually increase overall code readability and consistency.

Some words couldn't be replaced because they refer to locations and tools outside the project.

### CODE REVIEW

**Comment by marxin** [comment]
I'm personally fine with the changes, but I would like to hear opinions from other project maintainers.
Please fix testsuite fallout, run `pytest`.

File: rpmlint/checks/PAMModulesCheck.py [hunk 1]
```diff
@@ -8,7 +8,10 @@ class PAMModulesCheck(AbstractCheck):
 
     def __init__(self, config, output):
         super().__init__(config, output)
-        self.pam_whitelist = config.configuration['PAMModulesWhiteList']
+        self.pam_authorized_modules = config.configuration['PAMAuthorizedModules']
+        if not self.pam_authorized_modules:
+            # compatibility: read the values from the original configuration option
```
Comment (marxin) [inline rpmlint/checks/PAMModulesCheck.py:13]: This is a functional change and should go in a separate pull request.

**Review by marxin — CHANGES_REQUESTED** [review]

File: rpmlint/checks/PAMModulesCheck.py (hunk 1 above)
Comment (dmach) [reply rpmlint/checks/PAMModulesCheck.py:13]: ok, I'll split the commits into 2 pull requests.

Comment (marxin) [reply rpmlint/checks/PAMModulesCheck.py:13]: May I please remind this.

**Comment by Conan-Kudo** [comment]
I'm totally in favor of the changes, even if they're breaking things, as long as @dmach fixes the APIs/tests. 👍🏾 

This is rpmlint 2.0 after all, and it's not released yet. 💯

**Comment by marxin** [comment]
Merged, I'm going to revert the requested hunk.
